lib/config.js

```diff
@@ -64,30 +64,41 @@ function loadConfig(filePath) {
 }
 
 /**
+ * Loads plugin from node_modules or cache
+ * @param {string} pluginNameWithoutPrefix Name of the plugin without prefix
+ * @returns {Object} Plugin definition
+ */
+function loadPlugin(pluginNameWithoutPrefix) {
```

Comment:
Not command line config, *inline config* using `/*eslint-env*/` inside of code. What you have here doesn't work at all for that case.